src/Flowdock/Formatter/FlowdockMessageFormatter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Flowdock\Formatter;
+
+use Monolog\Formatter\NormalizerFormatter;
+
+final class FlowdockMessageFormatter extends NormalizerFormatter implements FlowdockFormatterInterface
```

Comment:
Do we need to extend the `NormalizerFormatter` ?

Reply:
Not really